offchain/hello-world-cli/test/Main.purs

```diff
@@ -142,6 +142,16 @@ main = do
               (cli <> "-c bad_path -s" <> state <> "query")
               "[Error: ENOENT: no such file or directory, open 'bad_path']"
       describe "integration test" do
+        when usePlutip $ it "server wallet works"
```

Comment:
*HSM

Reply:
AFAIU @marijanp wants to give it a proper review too so you probably shouldn't merge before that.